Library/Homebrew/utils/analytics.rb

```diff
@@ -16,64 +16,53 @@ class << self
 
       include Context
 
-      def report(type, metadata = {})
+      BUCKET = "analytics"
+      TOKEN = "bHDUBcO8rp521Y5lzCV4nrhWaYOeV-OjGeDVxv37m9Ee75CwTjir4EQr1hCHNJaEeRowiAjw3uSh9jG1Usmqfw=="
+      HOST = "europe-west1-1.gcp.cloud2.influxdata.com"
```

Comment:
Probably want to stick this behind a `.brew.sh` DNS so we can change it without needing to change the code.

Reply:
That makes sense, it's very early though. I still have to do a lot of testing with their cloud version.